packages/mds-policy-service/service/helpers.ts

```diff
@@ -26,7 +26,9 @@ export function translateIntentToPolicy<I extends INTENT_TYPE>(draft: IntentDraf
     draft.policy_fields.start_date = now() + TWENTY_MINUTES + POLICY_START_DATE_FUDGE_FACTOR
   }
 
-  const rules = asArray(draft.rule_fields).map(rule_field => ruleFieldToRule(draft.intent_type, rule_field))
+  const rules = Array.isArray(draft.rule_fields)
+    ? draft.rule_fields.map(rule_field => ruleFieldToRule(draft.intent_type, rule_field))
+    : [ruleFieldToRule(draft.intent_type, draft.rule_fields)]
```

Comment:
Why not use `asArray` here like we did previously? I think the behavior would be the same as what's being achieved in this ternary